libc-test/build.rs

```diff
@@ -1455,6 +1455,7 @@ fn test_dragonflybsd(target: &str) {
 
 fn test_wasi(target: &str) {
     assert!(target.contains("wasi"));
+    let p2 = target.contains("p2");
```

Comment:
```suggestion
    let p2 = target.contains("wasip2");
```

Just to make this a bit more clear